specs/results/results.spec

```diff
@@ -5,7 +5,7 @@ Precondition:
 * Validate Operator should be installed
 
 ## Test Tekton results with TaskRun: PIPELINES-26-TC01
-Tags: results, e2e, taskrun
+Tags: results, e2e, taskrun, sanity
```

Comment:
Let's keep sanity test suite minimal, I would only add the tag to the other scenario (for pipeline)